modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DELinkToPageField.testcase

```diff
@@ -160,12 +160,59 @@ definition {
 	}
 
 	@description = "This is a stubbed description"
-	@ignore = "true"
 	@priority = "5"
 	test CanInputDataOnDuplicatedField {
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Page Name 1");
+
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Page Name 2");
```

Comment:
@guivilarim, it's a best practice when writing a test to remove the necessary resources created to a test run, like this page `Test Page Name`, after all assertions of the test being created. Is this page being deleted after this test run? If not, we should do it.

In addition, do you really need to create 2 pages or use a single one in both fields Link to Page is enough?